python-project-template/src/{{module_name}}/__init__.py.jinja

```diff
@@ -1 +1,3 @@
-{% if create_example_module %}from .example_module import *{% endif %}
+{%- if create_example_module %}
+from .example_module import *
+{% endif %}
```

Comment:
There was a pylint error that was produced when there were too many blank lines in this file when the user requested that no example module was produced. 

This jinja vertical whitespace control seems to resolve that problem.

Reply:
The hydrated version of this file when the user does request an example module, looks like this: 

```

from .example_module import *

```

The leading blank line, makes Black unhappy.

Reply:
Seems like it was actually the `-` at the end of the if statement that was actually important...

Reply:
I confirmed this worked as expected for the following 4 cases: 

1. pylint and example module
2. pylint and no example module
3. black and example module
4. black and no example module